server/app/services/email/EmailSendMetrics.java

```diff
@@ -0,0 +1,47 @@
+package services.email;
```

Comment:
(blocking) Please move this to `services.monitoring` so it can leave next to the other metrics class.
(praise) Thank you for pulling this into a separate class!!!!!

Reply:
Thank you! Good call!! And thank you for starting the pattern :)